explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -645,15 +645,15 @@ export const buildTripleColumnGrid = (): React.ComponentProps<
  */
 function calculateEstimatedCellCount(
```

Comment:
Formatting of cell count has been removed from the function `calculateEstimatedCellCount` to `getEstimatedCellCount`.